setup.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python
-# Copyright 2015 SKA South Africa (http://ska.ac.za/)
+# Copyright (c) 2015-2019, National Research Foundation (Square Kilometre Array)
```

Comment:
We  should leave out the upper bound on the copyright, otherwise it has to be updated every year.  And might as well use our organisation's new name:
`# Copyright (c) 2015, National Research Foundation (South African Radio Astronomy Observatory)`

Reply:
Also think the copyright snippets in all the files need to be the same, so the others need to change to NRF (SARAO) - but they keep whatever initial year they had.